src/main/drivers/pwm_output.c

```diff
@@ -311,13 +311,14 @@ void motorDevInit(const motorDevConfig_t *motorConfig, uint16_t idlePulse, uint8
 #endif
 
         /* standard PWM outputs */
-        const unsigned pwmRateHz = useUnsyncedPwm ? motorConfig->motorPwmRate : ceilf(1 / (sMin + sLen * 2));
+        // margin of safety is 4 periods when unsynced
+        const unsigned pwmRateHz = useUnsyncedPwm ? motorConfig->motorPwmRate : ceilf(1 / ((sMin + sLen) * 4));
```

Comment:
@blckmn : If the reason is jitter compensation, it would be better to enter maximum tolerable jitter into calculation.
For ONESHOT125 this will set update rate to 1kHz, wasting quite a lot of bandwidth..